uxarray/reader/_scrip.py

```diff
@@ -0,0 +1,78 @@
+# Create list of names that can be used for node coords
+_X_NODES = ("lat", "latitude", "Lat", "Latitude", "X", "x", "grid_corner_lat",
+            "grid_corner_latitude")
+
+_Y_NODES = ("lon", "longitude", "Lon", "Longitude", "Y", "y", "grid_corner_lon",
+            "grid_corner_longitude")
+
+
+def viewkeys(d):
+    """Return either the keys or viewkeys method for a dictionary.
+
+    Args:
```

Comment:
For doc strings let us use the following format for consistency with Pangeo community (This applies to other doc strings throughout):

```
Parameters
----------

d : :obj:`dict`
        Dictionary that holds ... .

... other parameters if any...

Returns
--------

func() : 
    Description of func()'s purposes.
```